docs/getstarted/quick_start.md

```diff
@@ -125,23 +230,97 @@ $ floyd logs -t alice/quick-start/1
 ...
 ```
 
-The output of your code is printed in the `Run Output` section of the logs, between the `#########` lines. Anything you log or print in your code will appear here, so this is a great way to monitor the progress of your model training command. In our `quick-start` project, we're logging the Training Accuracy of our model.
+The output of your code is printed in the `Run Output` section of the logs,
+between the `#########` lines. Anything you log or print in your code will
+appear here, so this is a great way to monitor the progress of your model
+training command. In our `quick-start` project, we're logging the training
+accuracy of our model.
 
 Using the `-t` (tail) flag will stream the logs as they are generated.
 
-You can also view the logs in your browser using your `Job URL`. However, the logs in the Dashboard are not currently refreshed dynamically, so you'll need to refresh your browser periodically or press `F5` to get the latest logs from the Dashboard.
+You can also view the logs in your browser using your `Job URL`. However, the
```

Comment:
This is no longer true :) Logs auto-refreshed when job is running. You can remove this paragraph